prime-router/build.gradle.kts

```diff
@@ -852,7 +852,7 @@ dependencies {
     implementation("org.yaml:snakeyaml:2.2")
     implementation("io.github.linuxforhealth:hl7v2-fhir-converter") {
         version {
-            branch = "master"
+            branch = "platform/jwang/13020-runtime-constructors"
```

Comment:
Flip this back to master when changes to the HL7 to FHIR converter have been merged